src/murmur/Meta.cpp

```diff
@@ -282,6 +282,27 @@ void MetaParams::read(QString fname) {
 		}
 	}
 
+	QString qsVoiceProtocol = qsSettings->value("voiceProtocolPreference", QString()).toString();
+	if (! qsVoiceProtocol.isEmpty()) {
+#if QT_VERSION >= QT_VERSION_CHECK(5, 14, 0)
```

Comment:
Think about that! The Qt version used by the Linux CI and macOS CI is different!

Reply:
And the one used by the windows CI is different once again xD

Reply:
I hope this situation would be fixed after the cmake commit...

Reply:
Why would that be fixed? If everyone used the version provided by vcpkg then yeah. But that won*t be the case. On Linux for instance I think most people will continue the to use the Qt version shipped with their package manager (and on Macs folks will probably stay with homebrew)  :shrug: